src/Nethermind/Nethermind.Blockchain/IGenesisPostProcessor.cs

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core;
+
+namespace Nethermind.Blockchain;
+
+public interface IGenesisPostProcessor
+{
+    void PostProcess(Block genesis);
```

Comment:
potentially useful to pass also world state?
Or better leave it and pass through constructor as is?

Reply:
I think its fine as is.